inspectit-oce-demo/grafana/provisioning/elastic/dashboards/self-monitoring.json

```diff
@@ -0,0 +1,2096 @@
+{
```

Comment:
We have added a new component to inspectit with a new working-queue of classes with the following emtric:
inspectit_self_instrumentation_hooking_queue_size
You should use the maximum of this and the instrumentation queue size to compute the progress.